### ISSUE TITLE
prisma-test-utils is temporarily unmaintained

### ISSUE THREAD

**Issue by schickling** [prisma/prisma-test-utils#6]
While focusing on the release of production-ready version of Prisma 2 (see [blog post](https://www.prisma.io/blog/state-of-prisma-2-december-rcrwcqyu655e/)), we unfortunately don't have enough internal capacity to keep up the development of `prisma-test-utils` at the current point.

However, we're still very committed to making `prisma-test-utils` a first class citizen in the Prisma ecosystem and hope we'll have enough capacity again early 2020 to develop it further.  🚀 

Thanks a lot also to @maticzav for his great prior work on this project! 🙏

**Comment by ctrlplusb** [comment]
As discussed within slack...

I have created an example repo showcasing how to configure Jest to concurrently execute against Prisma + Postgres configuration where each test suite has its own db context spun up and down.

https://github.com/ctrlplusb/prisma2-pg-jest

Might be a good reference to some. 👍

**Comment by guledali** [comment]
@schickling Now that Prisma2 is out maybe this should be prioritized again? 
> Having a proper testing framework around **prisma** completes it.

**Comment by Aweller95** [comment]
@schickling Any updates regarding the re-maintaining of `prisma-test-utils` ?

**Comment by janpio** [comment]
If you have a clear need for this, please open an issue over at https://github.com/prisma/prisma and describe that use case so we can prioritize it.

**Comment by alexandrulesi** [comment]
Any updates?

**Comment by janpio** [comment]
No, my previous message https://github.com/prisma/prisma-test-utils/issues/6#issuecomment-706287410 still stands - right now we focus on other areas as we think those are more important.
